custom_model_runner/datarobot_drum/drum/runtime.py

```diff
@@ -27,14 +29,20 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
         if not exc_type:
             return True  # no exception, just return
 
+        if not self.options:
+            # exception occurred before args were parsed
+            return False  # propagate exception further
+
         # log exception
         exc_msg_lines = traceback.format_exception(exc_type, exc_value, exc_traceback)
         exc_msg = "".join(exc_msg_lines)
-        logger.error(exc_msg)
 
-        if not self.options:
-            # exception occurred before args were parsed
-            return False  # propagate exception further
+        if self.options.show_stacktrace:
+            logger.error(exc_msg)
+        else:
+            if exc_type == DrumCommonException:
+                print(exc_value)
+            exit(1)
```

Comment:
why we exit here?
if user passes `--with-error-server`, if exception is not `DrumCommonException` - the server will not be started

IMO it's better to:
```
if run_mode == RunMode.SERVER or self.options.show_stacktrace:
    logger.error(exc_msg)
```
-- we check is drum is run in server mode or if user passes `--show-stacktrace`, and if so - log error